components/config/definition.rst

```diff
@@ -13,7 +13,7 @@ Component. Configuration values are usually expected to show some kind of
 hierarchy. Also, values should be of a certain type, be restricted in number
 or be one of a given set of values. For example, the following configuration
 (in YAML) shows a clear hierarchy and some validation rules that should be
-applied to it (like: "the value for ``auto_connect`` must be a boolean value"):
+applied to it (like: "the value for ``auto_connect`` must be a Boolean value"):
```

Comment:
still goes wrong here, @xabbuh (seems like all "boolean" occurences are replaced with "Boolean" and all "Boolean" with "boolean")